From 29b3c8af37148072fadfc8c17a6ed1402989f412 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:51:40 -0400
Subject: [net] sunrpc: Remove using of dprintk with KERN_WARNING

Message-id: <1403560314-32525-159-git-send-email-bfields@redhat.com>
Patchwork-id: 84585
O-Subject: [Patch RHEL7 158/172] SUNRPC/NFSD: Remove using of dprintk with KERN_WARNING
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

When debugging, rpc prints messages from dprintk(KERN_WARNING ...)
with "^A4" prefixed,

[ 2780.339988] ^A4nfsd: connect from unprivileged port: 127.0.0.1, port=35316

Trond tells,
> dprintk != printk. We have NEVER supported dprintk(KERN_WARNING...)

This patch removes using of dprintk with KERN_WARNING.

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: a48fd0f9f77b6e144813230c26621c00aac92ce8
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfsfh.c b/fs/nfsd/nfsfh.c
index a337106..ec83934 100644
--- a/fs/nfsd/nfsfh.c
+++ b/fs/nfsd/nfsfh.c
@@ -88,9 +88,8 @@ static __be32 nfsd_setuser_and_check_port(struct svc_rqst *rqstp,
  /* Check if the request originated from a secure port. */
  if (!rqstp->rq_secure && !(flags & NFSEXP_INSECURE_PORT)) {
   RPC_IFDEBUG(char buf[RPC_MAX_ADDRBUFLEN]);
-  dprintk(KERN_WARNING
-         "nfsd: request from insecure port %s!\n",
-         svc_print_addr(rqstp, buf, sizeof(buf)));
+  dprintk("nfsd: request from insecure port %s!\n",
+          svc_print_addr(rqstp, buf, sizeof(buf)));
   return nfserr_perm;
  }
 
diff --git a/net/sunrpc/svcsock.c b/net/sunrpc/svcsock.c
index 74138ab..92957de 100644
--- a/net/sunrpc/svcsock.c
+++ b/net/sunrpc/svcsock.c
@@ -849,8 +849,7 @@ static struct svc_xprt *svc_tcp_accept(struct svc_xprt *xprt)
   * tell us anything.  For now just warn about unpriv connections.
   */
  if (!svc_port_is_privileged(sin)) {
-  dprintk(KERN_WARNING
-   "%s: connect from unprivileged port: %s\n",
+  dprintk("%s: connect from unprivileged port: %s\n",
    serv->sv_name,
    __svc_print_addr(sin, buf, sizeof(buf)));
  }
-- 
1.7.1